ext/session/session.c

```diff
@@ -2711,6 +2723,22 @@ static const zend_function_entry session_functions[] = {
 
 /* {{{ SessionHandlerInterface functions[]
 */
+static const zend_function_entry php_session_save_handler_iface_functions[] = {
+	PHP_ABSTRACT_ME(SessionSaveHandlerInterface, open, arginfo_session_class_open)
```

Comment:
@Ocramius 
Since you insists validateId() is not needed, you seems you don't understand the reasons why session.use_strict_mode=1 is important. Please research why this is important.